src/client/game.cpp

```diff
@@ -595,15 +594,16 @@ struct GameRunData {
 	u16 new_playeritem;
 	PointedThing pointed_old;
 	bool digging;
-	bool ldown_for_dig;
+	bool btn_down_for_dig;
 	bool dig_instantly;
 	bool digging_blocked;
-	bool left_punch;
+	bool punch;
+	bool update_wielded_item_trigger;
```

Comment:
It appears this line was not an addition in the original PR?  Just curious if it was taken out at some point.  Good luck, I hope you can get this to work again.  :)

Reply:
This line seems to have already existed in the code, before the original PR, but was removed in a later commit. This would've been added during rebase. I still kinda suck at rebasing in all but the simplest of situations :)